pkg/render/utils.go

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2023 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package render
+
+import (
+	"fmt"
+
+	operatorv1 "github.com/tigera/operator/api/v1"
+)
+
+// LinseedNamespace determine the namespace in which Linseed is running.
+// For management and standalone clusters, this is always the tigera-elasticsearch
+// namespace. For multi-tenant management clusters, this is the tenant namespace
+func LinseedNamespace(tenant *operatorv1.Tenant) string {
```

Comment:
Any reason not to put this in the `linseed` package?

Reply:
```
package github.com/tigera/operator/pkg/render/logstorage/linseed
	imports github.com/tigera/operator/pkg/render
	imports github.com/tigera/operator/pkg/render/logstorage/linseed: import cycle not allowed
```